react-native-libraries.json

```
@@ -8823,5 +8823,21 @@
     "android": true,
     "web": true,
     "expo": true
-  }
+  },
+  {
+    "githubUrl": "https://github.com/onmotion/react-native-autocomplete-dropdown",
+    "npmPkg": "@onmotion/react-native-autocomplete-dropdown",
+    "examples": [
+        "https://github.com/onmotion/react-native-autocomplete-dropdown/tree/main/example/components"
+    ],
```

```suggestion
    "examples": [
        "https://snack.expo.io/@onmotion/react-native-autocomplete-dropdown",
        "https://github.com/onmotion/react-native-autocomplete-dropdown/tree/main/example/components"
    ],
```

You can also add Snack to the examples. 🙂

---

Oh okay, got it